content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas (Tasks)
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (Dasbor)
+
+Melakukan _deploy_ dan mengakses dasbor (_dashboard_) berbasis web untuk 
```

Comment:
```suggestion
Melakukan _deploy_ dan mengakses Dashboard berbasis web untuk 
```
same as above

Reply:
I think it should lowercase because it does not belong to Kubernetes resource name.